Protocol.md

```diff
@@ -162,6 +162,51 @@ responders, the server MUST dynamically assign identifiers (in the range
 of `0x02..0xff`). A server-assigned address becomes invalid as soon as
 the connection to the server has been severed.
 
+# Exchanging Signalling Information
+
+Please note that this section is informational only.
+
+In order to establish a signalling channel using SaltyRTC, the following
+information has to be available to both peers:
+
+* WebSocket URI scheme (`ws` or `wss`),
```

Comment:
Ah: *Although the SaltyRTC protocol takes many security measures to prevent eavesdropping, it is still highly RECOMMENDED to use WebSocket in its secure mode*